### PR TITLE
chore(tsdocs): move fixture build out of main build task

### PR DESCRIPTION
**Pull request by bajtos**
Rework tsdocs build scripts as follows:

Introduce a new npm task `build:fixtures` that can be used to rebuild files in `fixtures/monorepo`.

Move this new task out of the main `build` task for the following
reasons:

1. The global `npm run build` command no longer invokes `build` scripts in individual packages, we use Project References to build the entire loopback-next monorepo as a whole. I find it confusing to have different behavior when a package is built from monorepo or locally.

2. It is very rare that we need to (re)build tsdoc fixtures. The files created by the compiler are versioned by `git`, rebuild is needed only to update JavaScript files to match the new `tsc` output.

3. Sometimes, I would see tsdoc fixtures build triggered by unrelated actions, e.g. `npm run tsdocs`, which adds unnecessary friction to our dev process.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/tsdocs/package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "extract-apidocs": "node bin/extract-apis",
     "document-apidocs": "api-documenter markdown -i ../../docs/apidocs/models -o ../../docs/site/apidocs",
     "update-apidocs": "node bin/update-apidocs",
-    "build": "lb-tsc && npm run build --prefix fixtures/monorepo",
+    "build": "lb-tsc",
+    "build:fixtures": "npm run build --prefix fixtures/monorepo",
```
Comment (raymondfeng) [inline packages/tsdocs/package.json:17]: Do we want to run it as part the `npm run build:full` for `loopback-next`? I'm worried that we forget to rebuild the fixtures during TypeScript upgrade.

Comment (bajtos) [reply packages/tsdocs/package.json:17]: Sure, why not. 

I feel such change is out of scope of this pull request though.

In this PR, I am removing the inconsistency where `npm run build` in monorepo does not build tsdocs fixtures while `npm run build` in `packages/tsdocs` does.

I'd like to land this PR first, and then discuss what process we want to use to ensure fixtures are rebuilt after TypeScript upgrades.

One option that comes to my mind is to re-build the fixtures from `postinstall` hook, which is honored by `lerna bootstrap` and is not executed as often as the build task.

File: packages/tsdocs/package.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
...
     "clean": "lb-clean loopback-tsdocs*.tgz dist *.tsbuildinfo package",
     "pretest": "npm run build",
```
Comment (raymondfeng) [inline packages/tsdocs/package.json:19]: Let's add `npm run build:fixtures` to `pretest`.

Comment (bajtos) [reply packages/tsdocs/package.json:19]: I don't like the idea of rebuilding fixtures before every test run. I want the tests to finish as quickly as possible, adding a build step that's a no-op in most times is not optimal.

**Comment by bajtos** [comment]
Opened a follow-up PR: https://github.com/strongloop/loopback-next/pull/5767

**Pull request merged**